github actions based clicklist updater

theofficialgman opened the pull request:
closes https://github.com/Botspot/pi-apps-analytics/issues/5

I still need the current `linklist` as that file will need to be stored in the github as well (or as a variable in secrets I guess might also work). I assume this is a list of the links for apps that have been made on bitly

this does not work as is

theofficialgman committed:
github actions based clicklist updater

Botspot commented:
[linklist.txt](https://github.com/Botspot/pi-apps-analytics/files/7902032/linklist.txt)

theofficialgman committed:
add linklist and update scripts

theofficialgman committed:
fix export

theofficialgman committed:
add secrets (testing)

theofficialgman committed:
update applist

repo_url doesn't exist (anymore?) so either need to get appslist ourself or download to the update folder

easier to just get the applist ourself

theofficialgman committed:
rename linklist

theofficialgman committed:
swap to put secrets in " "

theofficialgman committed:
comment out "Push changes to repo"
